lib/provisioner/worker/plugins/automators/docker_automator/docker_automator.rb

```diff
@@ -106,9 +106,41 @@ def portmap
     portmap
   end
 
+  def envmap
+    envmap = ''
+    # TODO: allow commas inside quotes
+    @fields['environment_variables'].split(',').each do |env|
```

Comment:
method body can just be ``@fields['environment_variables'].split(',').map {|x| "-e #{x}" }.join(' ')``.  it's a bit cleaner and wont leave the extra trailing space